src/test/java/AddAndNotifyCommandTest.java

```diff
@@ -0,0 +1,32 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class AddAndNotifyCommandTest {
+    @Test
+    void notificationMustBeSentToEmailThatWasEnteredByUser() {
```

Comment:
Okey, Happy case is tested. What about fail case ? Just turn off internet on your computer  and try to execute the command via Main method